.gitignore

```diff
@@ -140,5 +140,3 @@ iter8
 **/ghz-call-data.json
 **/ghz-call-data.bin
 **/ghz-call-metadata.json
-
-**/load-test-http/
```

Comment:
why just this one? what about load-test-grpc?

Reply:
The PR actually removes that line. Sometimes, when running tests, we pull down load-test-http and pollute other folders. We can selectively ignore load-test-http in those test folders if need be.

Reply:
I'm blind.